rfcs/text/0000-Managed-Access-Data.md

```diff
@@ -0,0 +1,346 @@
+for specific research use. This can happen for many reasons including
+the national law where they are based or the types of individuals they
+are collecting data from e.g individuals with a disease. This means
+there is a growing set of data suitable for building the Human Cell
+Atlas (HCA) which can't currently be stored, processed and distributed
+by the Data Coordination Platform (DCP).
+
+The processing of genetic and genomic information necessarily raises
+questions as to what information is 'identified or identifiable' (and
+thus subject to GDPR) and what information is 'anonymous or anonymized'
+(and thus not subject to GDPR). These difficulties are both legal (for
+the purposes of determining GDPR's scope of application) and scientific
+-- for the purposes of determining what data is sensitive for data
+subjects, for the purposes of determining what information the data can
+reveal about the data subject (either alone or in combination with other
+datasets). Scientific difficulties also inhere, for the purposes of
+determining what external data can be combined with the data subject's
+data to reveal their identity.
+
+This proposal will describe the architectural components and the
+security compliance requirements necessary for the DCP to be approved as
+an archive for controlled access data from a variety of nations. There
+are specific guidelines that must be met before controlled access data,
+especially US government research data, can be stored onto the system.
+The safe storage rules for US data is different from the rules in other
+countries (though there is a lot of overlap). Each set of rules will
+have specific requirements that must be met. This RFC will also outline
+the steps required to meet basic security compliance requirements
+necessary before controlled access data can be stored in the system.
+
+### **Definitions**
+
+***Managed access data*** - Raw and processed genomic and phenotypic
+data from individuals whose consent agreements authorise data release
+only for specific research use to bona fide researchers.
+
+***Open access data*** - Raw and processed genomic and phenotypic data
+from individuals whose consent agreement authorise unrestricted sharing
+and summary level data generated from both open and managed access raw
+and processed genomic data.
+
+***HCA Data Contributor Agreement*** - an agreement entered into between
+the Data Contributor and the HCA, prior to contribution and upload of
+data to the HCA DCP.
+
+***Data Access Committee (DAC) -*** A committee responsible for deciding
+who should be able to access data from a study, and what use the
+researcher can make of that data. Each researcher submits a request of
+who they are, which data they want, and what they will use it for. Each
+type of use requires a separate request to the DAC.
+
+### **User Stories**
+
+As a Researcher with a keyboard, I want to find data that is similar to
```

Comment:
Can you number these to make them easier to refer.

Reply:
yes, thanks.